src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -1488,7 +1488,7 @@ private static LambdaConversionResult IsAnonymousFunctionCompatibleWithDelegate(
                 {
                     for (int p = 0; p < delegateParameters.Length; ++p)
                     {
-                        if (!OverloadResolution.AreRefsCompatibleForMethodConversion(delegateParameters[p].RefKind, anonymousFunction.RefKind(p), compilation) ||
+                        if (!OverloadResolution.AreRefsCompatibleForMethodConversion(source: anonymousFunction.RefKind(p), destination: delegateParameters[p].RefKind, compilation) ||
```

Comment:
>source: [](http://example.com/codeflow?start=85&length=7)

Naming of parameters feels confusing to me. Data flow from delegate parameter to the lambda parameter. So, I would expect the delegate parameter to be the "source".